peepingtom/datablocks/multiblocks/particleblock.py

```diff
@@ -16,21 +16,13 @@ class ParticleBlock(OrientedPointBlock):
         'class_plot': ClassPlotDepictor,
```

Comment:
ClassPlot is for plotting particles in different colours according to their class, right? 
 
In general I think names should be a little more descriptive e.g. `ClassificationResultsDepictor`

I've also been thinking about removing depictor from the names of these classes, I don't think we need it, what's your feeling on that?

Reply:
Yes, we can change this to clearer names. I would keep Depictor in it though, to be consistent (and since we can't remove `Depictor` from `ParticleDepictor` without being very ambiguous).